_posts/2016-07-14-polyrhythms.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "It's Like Patting Your Head and Rubbing Your Tummy: Polyrhythms"
+author: "Hayley MacLeod"
+categories: music
+header: /assets/img/2016-07-15/hypnosis.jpg
+---
+
+So you want to get better at rhythms, eh? Gain a little independence between your hands? Polyrhythms are a great way to start. A polyrhythm is two different rhythmic patterns played simultaneously against one another.
+
+With your attention on the individual beats within the two rhythms, you become entranced by the resulting pattern. Polyrhythms are known for their hypnotic character. It may be why we enjoy listening to them as much as we want to get better at playing them.
```

Comment:
Assuming the reader completely understands based off of the single sentence explanation above. Better to actually describe the aural effect polyrhythms create to further understanding rather than describing an emotional bond. Or use the aural effect to explain the emotional bond instead of just stating a subjective stance.